Views/Content-CarouselItem.liquid

```diff
@@ -1,13 +1,13 @@
-﻿{% assign linkedContent = Content.ContentItemId[Model.ContentItem.Content.CarouselItem.CTALinkTo.ContentItemIds[0]] %}
-
-<div class="card card--carousel">
-    <div class="card__body">
-        <h2 class="text--size-h1">{{ Model.ContentItem.Content.CarouselItem.Heading.Text | raw }}</h2>
-
-        {% if linkedContent %}
-            <p><a class="btn btn--secondary" href="{{ linkedContent | display_url }}">{{ Model.ContentItem.Content.CarouselItem.CTALabel.Text | raw }}</a></p>
-        {% endif %}
-    </div>
-
-    {{ Model.Background | shape_render }}
-</div>
\ No newline at end of file
+﻿{% assign linkedContent = Content.ContentItemId[Model.ContentItem.Content.CarouselItem.CTALinkTo.ContentItemIds[0]] %}
+
+<div class="card card--carousel">
+    <div class="card__body">
+        <h2 class="text--size-h1">{{ Model.ContentItem.Content.CarouselItem.Heading.Text | raw }}</h2>
+
+        {% if linkedContent %}
+            <p><a class="btn btn--secondary" href="{{ linkedContent | display_url }}">{{ Model.ContentItem.Content.CarouselItem.CTALabel.Text | raw }}</a></p>
+        {% endif %}
+    </div>
+
+    {{ Model.Background | shape_render }}
+</div>

```

Comment:
It's probably time for us to have a conversation about whether carousel should really use card as it's base block. 

I'd be inclined to suggest that carousel should become it's own BEM block and that the styles currently in card/carousel become carousel/default... Would others agree, or am I on my own with that? 😂 

cc @bengammon for opinions

Reply:
@Newnab Completely agree, I didn't like that half the styles for the carousel component were in the `card` component.

Reply:
Sorted.